src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -354,6 +354,7 @@ def update_container_branches() {
         do
             echo \$repo
             cd ${env.WORKSPACE}/ci/\$repo
+            eups declare -r . -t current
```

Comment:
I strongly suggest calling `eups declare` *after* checking out the branche, in case anything important happens on the branch that affects the behavior of `eups declare`. I don't know what that might be, but why take chances?